core/tx_multicurrency_priceheap_test.go

```diff
@@ -305,7 +313,7 @@ func TestIsCheaper_FwdFields(t *testing.T) {
 		assert.Equal(t, curr2, *c2)
 		return -1
 	}
-	assert.True(t, cmp.IsCheaper(tx1, tx2))
+	assert.True(t, cmp.Cmp(tx1, tx2, nil, nil) <= 0)
 }
 
 func TestIsCheaper(t *testing.T) {
```

Comment:
This test doesn't test anything. Assuming that `CmpValues` is covered here, we need to test the `(cc CurrencyCmpFn) Cmp` method, making sure that TXs are compared properly in different situations.

Reply:
This tests only checks that the Cmp fn forwards the values in the proper order. I'll add a new test for comparing txs (independently from the multicurrency tests)

Reply:
> This tests only checks that the Cmp fn forwards the values in the proper order.

What do you mean exactly?

Reply:
That the fields to Cmp arrived in the proper order, but I deleted it since it doesn't add much.

Reply:
Ah, that must be a misunderstanding them, I meant the `TestIsCheaper` test. Since it's not testing `IsCheaper` and the comparison functions are mocked, it just tested that -1 <= 0, 0 <= = and 1 >= 0, so I thought we could remove it.

Reply:
That one is testing that the return value from the Cmp function is being returned successfully to the IsCheaper function.

Reply:
How does it do it if there is no `IsCheaper` call?

Reply:
Sorry, meant returning from the currency comparison to the transaction comparison